src/logics/iolend/tokens/index.ts

```diff
@@ -0,0 +1,6 @@
+import * as common from '@protocolink/common';
+import iotaTokensJSON from './data/iota.json';
+
+type IotaTokenSymbols = keyof typeof iotaTokensJSON;
+
+export const iotaTokens = common.toTokenMap<IotaTokenSymbols>(iotaTokensJSON);
```

Comment:
```suggestion
export const iotaTokens = { ...common.toTokenMap<IotaTokenSymbols>(iotaTokensJSON), ...common.iotaTokens };
```

We could merge custom tokens and common tokens here and use them in `*.test.ts`, just like we currently do with aave-like.

Reply:
I think it's okay not to merge the aTokens since we can easily recognize where to find the tokens. Also, the aTokens in iota logics do not need to be replaced.

Reply:
okay. no problem!